### DISCUSSION TITLE
how i commit my project on github using lvim

### DISCUSSION THREAD

**Discussion by Mohita9138** [LunarVim/LunarVim#3804] [Q&A] [answered]
### Problem description

can any one tell me how to commit my project on github using lvim

### LunarVim version

master-c18cdf0

### Neovim version (>= 0.8.0)

0.8.2

### Operating system/version

kali linux

**Comment by brneor** [comment, marked as answer]
You can use [lazygit](https://github.com/jesseduffield/lazygit) with [lazigit.nvim](https://github.com/kdheepak/lazygit.nvim) plugin, but there are a plethora of other options.

Also, you should use the Discussions area for this kind of question and ideally not use the `bug` label since this is not a bug but a general question.

**Comment by Mohita9138** [comment]
thanks bro im realy sorry for bug lable

**Comment by Mohita9138** [comment]
> You can use [lazygit](https://github.com/jesseduffield/lazygit) with [lazigit.nvim](https://github.com/kdheepak/lazygit.nvim) plugin, but there are a plethora of other options.
> 
> Also, you should use the Discussions area for this kind of question and ideally not use the `bug` label since this is not a bug but a general question.

can you tell me how to install this plugin in lvim

**Comment by brneor** [comment]
@Mohita9138 you can find the desired information [on the Lunarvim's docs](https://www.lunarvim.org/docs/plugins#user-plugins).
